History/js/new.js

```diff
@@ -249,6 +246,34 @@ var callbacks = {
       // Clear the old iframe and insert the new one
       $(".privly_iframe").empty();
       $(".privly_iframe").append(iFrame);
+    }
+
+    $('button#prev_preview').on('click',function(){
+      //finds the previous button with preview_link class and passes its reference.
+      //a small example of why this works : http://jsfiddle.net/5QdgB/
+      var index = $('.preview_link').index(buttonClicked);
+      var previous = $('.preview_link').slice(index-1).first();
+      console.log("previous = "+previous);
+      buttonClicked = previous;
+      iframeReturn(previous);
+    });
+
+    $('button#next_preview').on('click',function(){
```

Comment:
You should probably put comments above several of these event handlers to describe their purpose since they are multi-line functions.